solr/api/src/java/org/apache/solr/client/api/endpoint/NodeFileStoreApis.java

```diff
@@ -63,5 +64,6 @@ SolrJerseyResponse getFile(
           String getFrom,
       @Parameter(description = "Indicates that (only) file metadata should be fetched")
           @QueryParam("meta")
-          Boolean meta);
+          Boolean meta)
+      throws IOException;
```

Comment:
Just wanted to confirm this throw was needed?   Maybe because of something else furthur down?